deploy/eck-stack/charts/eck-elasticsearch/values.yaml

```diff
@@ -127,15 +127,15 @@ volumeClaimDeletePolicy: ""
 
 # Settings to limit the disruption when pods need to be rescheduled for some reason such as upgrades or routine maintenance.
 # Default budget selects all cluster pods and sets `maxUnavailable` to 1.
-# To disable, set to the empty value (`{}`).
+# To disable, set `disabled` to true.
```

Comment:
I think this now needs to be a little more clear, maybe. 

```
# By default when `podDisruptionBudget` is unset, or set to `null` the operator sets a default pod disruption budget and selects all cluster pods and sets `maxUnavailable` to 1.
# To have the operator set an empty pdb set to the empty value (`{}`).
# To completely disable the pdb set `disabled` to true.
```

Reply:
I see but the important part is that "if unset, the default is X" and it is not so important whether the default is set by Helm or the operator. But let's try to avoid this misunderstanding.

Do you think we should rewrite the others?

https://github.com/elastic/cloud-on-k8s/blob/53e7f25aac53a5a5f4340bf897ac1edfed27239d/deploy/eck-stack/charts/eck-elasticsearch/values.yaml#L124

https://github.com/elastic/cloud-on-k8s/blob/53e7f25aac53a5a5f4340bf897ac1edfed27239d/deploy/eck-stack/charts/eck-elasticsearch/values.yaml#L146

Reply:
Well, `revisionHistoryLimit` is a k8s default setting isn't it, not set by default by eck? I'm on the fence on adjusting that wording.

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#clean-up-policy

And `volumeClaimDeletePolicy` is definitely defaulted by ECK, so I'd support slightly adjusting the wording, sure.

Reply:
e25f5d7